### DISCUSSION TITLE
Cancellation of Nov 28 TODO Touchpoint Call & OSS in Business Guide Updates

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#522] [Announcements]
Please note that this Thursday’s TODO Touchpoint call is canceled. Over the past touchpoint calls, we have combined open office hours with discussions on the OSS in Business Guide initiative. During these sessions, the community has made significant progress, including:

- Secured commitment by the community to contribute to the creation of this guide. Discussions initiated below [this comment that shared the initial proposal](https://github.com/todogroup/ospology/discussions/481)
- Working Group approved by Steering Committee. [Here](https://github.com/todogroup/ospology/discussions/481#discussioncomment-10771567)
- Developing by the community participants an initial outline for focus areas. [Here](https://github.com/todogroup/ospology/discussions/481#discussioncomment-10157135)
- Working Repo Created, under [OSPOlogy](https://github.com/todogroup/ospology/tree/main/whitepapers/business-value)
- Sharing several presentations on the motivation and vision for this initiative by its initial leaders Thursday TODO Europe Touchpoints. [Here](https://github.com/todogroup/ospology/discussions/512#discussioncomment-11168620)
- Reviewing additional presentations as inspiration for the overall project during Thursday TODO Europe Touchpoints. [Here](https://github.com/todogroup/ospology/discussions/512#discussioncomment-11168620)

### Current Status

We have reached a stage where a roadmap with clear deadline expectations and in-depth contribution guidelines is needed, as these deliverables will assist working group participants streamline their contributions and use their time more efficiently during catch-up calls. 

Due to several challenges identified in the preparation for this week’s TODO Touchpoint call, the decision was made to cancel the meeting and focus on resolving key issues asynchronously. This pause will allow the community to address critical gaps, align efforts with related initiatives, and establish a stronger foundation for future collaboration. Below are the main reasons for this cancellation and the rationale behind the decision:

1. Limited progress on resources: Resources such as a roadmap and contribution guidelines, have not been sufficiently developed to support productive discussions during this week’s call
2. Need for cross-project discussions: Further coordination with adjacent OSS Readiness / OSPO initiatives across other LF projects that just started is needed to ensure efforts are aligned to enhance the impact of this work and ensure a cohesive approach across projects

### Next Steps

We aim to reconvene in January with:

- A new and more structured schedule for regular meetings 2025 series
- Documentation in the working repository, outlining:
    - Contribution guidelines
    - How to file issues and create content
    - Standardized processes for collaboration

### Updates

A couple of community volunteers have stepped forward to lead the project management efforts. An initial proposal is currently under review by the TODO LF Staff Project Manager to later be overseen by the TODO Steering Committee. Once the roadmap and processes are finalized (expected before year-end), all relevant details will be documented in the working repository and OSPO Forum.

We hope this temporary pause will allow participants to maximize their contributions while saving time, as well as share use cases more effectively with allocated time and dedicated roles in the community to manage organizational tasks, to get this initiative back on track.

Thank you for your understanding. More updates coming soon!

> To check next meetings, please take a look at our [Calendar](https://todogroup.org/community/meetings/#calendar)

**Comment by mkurzman** [comment]
Hi Ana, could you please also update the meeting info in https://github.com/todogroup/ospology/tree/main/whitepapers/business-value accordingly?
Thank you.
